addons/l10n_au/data/localisation.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+	<data noupdate="1">
+		<record id="base.AUD" model="res.currency">
+			<field name="position">before</field>
+			<field name="active" eval="True"/>
+		</record>
+
+		<record id="base.NZD" model="res.currency">
+			<field name="position">before</field>
+			<field name="active" eval="True"/>
+		</record>
+
+		<record id="base.JPY" model="res.currency">
+			<field name="active" eval="True"/>
+		</record>
+
+		<record id="base.CNY" model="res.currency">
+			<field name="active" eval="True"/>
+		</record>
+
+		<record id="base.MYR" model="res.currency">
+			<field name="active" eval="True"/>
+		</record>
+
+		<record id="base.SGD" model="res.currency">
+			<field name="active" eval="True"/>
+		</record>
+
+		<record id="base.GBP" model="res.currency">
+			<field name="active" eval="True"/>
+		</record>
+
+		<record id="base.lang_en" model="res.lang">
```

Comment:
@mart-e @gdgellatly 
Previously there was no AU, so great that has been added.  I am happy with it's definition, too.  I will drop this out.

@gdgellatly 
We have always loaded our clients with a tiny patched js that fixes that anomaly.
In version 9 (in the enterprise version at least), date entries are actually rejecting entry of 6/1/10!